services/movie_session.py

```diff
@@ -0,0 +1,50 @@
+from datetime import datetime
+
+from django.db.models import QuerySet
+
+from db.models import MovieSession
+
+
+def create_movie_session(
+        movie_show_time: datetime,
+        movie_id: int,
+        cinema_hall_id: int
+) -> QuerySet:
```

Comment:
The return type for `create_movie_session` is specified as `QuerySet`, but the `create` method returns an instance of `MovieSession`, not a `QuerySet`. Consider changing the return type to `MovieSession`.